frontends/web/src/routes/bitboxbase/bitboxbase.tsx

```diff
@@ -70,35 +86,46 @@ class BitBoxBase extends Component<Props> {
         {
             t,
             ip,
-            bitboxBaseIDs,
+            detectedBases,
         }: RenderableProps<Props>,
         ) {
             return (
             <div class="contentWithGuide">
                 <div class="container">
-                    <Header title={t('bitboxBase.title')} />
+                    <Header title={<h2>{t('bitboxBase.title')}</h2>} />
                     <div class="innerContainer scrollableContainer">
                         <div class="content padded">
+                            {
+                                detectedBases ?
+                                (
+                                    <div>
+                                        <h3>{t('bitboxBase.detectedBases')}</h3>
+                                        {
+                                            Object.keys(detectedBases).map(hostname => (
```

Comment:
since you need both key and value, mapping over both at once with `Object.entries(([hostname, ip]) => ... )`  is more suitable